src/split/split.cpp

```diff
@@ -219,9 +220,9 @@ Thank you!\n";
 		std::cerr << "\n\nUsage error: " << e.what() << ".\n";
 		return 1;
 	}
-	catch(parse_error& e) {
-		std::cerr << "\n\nParse error on line " << e.line << " in file \"" << e.file.string() << "\": " << e.reason << '\n';
-		return 1;
+	catch(pdbqt_parse_error& e) {
+		std::cerr << e.what();
```

Comment:
For being consistent with the rest of the error output this should probably be prefaced with "\n\nParse error: "